pkg/util/execdetails/execdetails.go

```diff
@@ -1542,15 +1542,8 @@ func (e *BasicRuntimeStats) GetActRows() int64 {
 }
 
 // Clone implements the RuntimeStats interface.
-func (e *BasicRuntimeStats) Clone() RuntimeStats {
-	result := &BasicRuntimeStats{}
-	result.executorCount.Store(e.executorCount.Load())
-	result.loop.Store(e.loop.Load())
-	result.consume.Store(e.consume.Load())
-	result.open.Store(e.open.Load())
-	result.close.Store(e.close.Load())
-	result.rows.Store(e.rows.Load())
-	return result
+func (*BasicRuntimeStats) Clone() RuntimeStats {
```

Comment:
It'll be better to add a comment for why clone should not be invoked for this func.

Reply:
Ok, add it.